### ISSUE TITLE
It’s Hard For Neural Networks to Learn the Game of Life

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#125]
# TL;DR
A study examining the dependence of neural networks on initial values in a simulation called Game of Life. Game of Life can be represented by a two-layer CNN because it uses relatively simple rules for updating, but in reality, the rules can only be learned correctly in a much larger network, and the larger the It learns robustly to initial perturbations. The experimental results support the lottery ticket hypothesis.

![Fraction successful trials 6](https://user-images.githubusercontent.com/24524018/95158531-c84fc980-07d6-11eb-9fbb-3727b1fec824.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.01398

# Submission Dates(yyyy/mm/dd)
2020/09/02

# Authors and institutions
Jacob M. Springer, Garrett T. Kenyon
- Swarthmore College
- Los Alamos National Laboratory

# Methods

# Results

# Comments
